src/Docker/Machine/DockerMachineCli.php

```diff
@@ -90,7 +90,12 @@ public function getIp()
      */
     public function isCreated()
     {
-        return $this->processRunner->run('docker-machine status '.$this->name, false)->isSuccessful();
+        $process = $this->processRunner->run('docker-machine status '.$this->name, false);
+        if(!$process->isSuccessful()) {
+            return false;
+        }
+
+        return strpos($process->getOutput(), 'Error') === 0;
```

Comment:
So... the machine is created only if the output of the process contains `Error` ?